site/en/r2/tutorials/keras/basic_regression.ipynb

```diff
@@ -85,17 +101,17 @@
         "id": "bBIlTPscrIT9"
       },
       "source": [
-        "\u003ctable class=\"tfo-notebook-buttons\" align=\"left\"\u003e\n",
```

Comment:
@lamberta do you know if these escapes are necessary?

Reply:
The escapes come from saving with internal Colab (vs public Colab). It's annoying